R/monitoring.R

```diff
@@ -365,7 +384,7 @@ monitor.matrix <- function(obj, X, y, verbose=TRUE){
     for(k in (obj$last+1):nrow(X)){
       newestims <- obj$computeEstims(X,y,k)
       obj$process <- rbind(obj$process,
-                           obj$computeEmpProc(newestims$coef, newestims$Qr12,k))
+                           obj$computeEmpProc(newcoef = newestims$coef, Q = newestims$Qr12, k = k))
```

Comment:
this feels like an unnecessary fix for the upstream.
No code is actually changed. It's only more verbose now.
In the spirit of staying as close as possible to strucchange code, perhaps we should reverse it to the previous, less explicit, code.
```suggestion
                           obj$computeEmpProc(newestims$coef, newestims$Qr12,k))
```